### PR TITLE
Scenario service html table headers from types

### PR DESCRIPTION
**Pull request by akalburgi**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/docs/source/support/release-notes.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs [hunk 1]
```diff
@@ -829,14 +829,24 @@ renderRow world parties NodeInfo{..} =
...
+        
 -- TODO(MH): The header should be rendered from the type rather than from the
 -- first value.
 renderTable :: LF.World -> Table -> H.Html
 renderTable world Table{..} = H.div H.! A.class_ active $ do
     let parties = S.unions $ map niObservers tRows
     H.h1 $ renderPlain $ prettyDefName world tTemplateId
-    let (headers, rows) = unzip $ map (renderRow world parties) tRows
-    H.table $ head headers <> mconcat rows
+    let (_, rows) = unzip $ map (renderRow world parties) tRows
```
Comment (neil-da) [inline compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs]: `renderRow` returns a pair of the header and the rows. We are now not using the header, so can we delete it from renderRow, and simplify this bit

File: compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs [hunk 2]
```diff
@@ -829,14 +829,24 @@ renderRow world parties NodeInfo{..} =
...
+    let headers = renderHeader world parties (head tRows)
+    H.table $ head [headers] <> mconcat rows
```
Comment (neil-da) [inline compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs]:
```suggestion
    H.table $ headers <> mconcat rows
```
`head [x] == x`

**Comment by akalburgi** [comment]
> Looks reasonable. What was the motivation behind this change? Can you confirm it results in no visible changes?
This was my initial attempt, and this not the intended outcome for this change. I am still working on it with @hurryabit

**Comment by akalburgi** [comment]
Closing this in favor of the other PR

**Pull request closed without merging**